### ISSUE TITLE
Cant use n-gram data using server, WIKI docs don't make sense

### ISSUE THREAD

**Issue by David-Else** [languagetool-org/languagetool#2803]
I have been through the WIKI article, but nothing works. It is not explained properly.

> Server mode: Start with the --languageModel option. Alternatively, you can start with the --config file option. This properties file needs to have a languageModel=… entry pointing to the ngram-index directory. Using the properties files will give you some advanced configurations. Call java -jar languagetool-server.jar to get a list of all options.

So there are two options:
1.  Start with the `--languageMode`l option
2.  Start with the `--config` file option

This implies I can just add `--languageModel` and it will work, which is does not. How about the file reference? It needs to point to a dir, but that is not mentioned. When it points to a dir it does not work.

Elsewhere in the manual it is stressed to point to the directory containing the n-grams, not the engram directory which should be named with `en`, or whatever language. This is not mentioned.

So my current failed attempt looks like this, I am afraid I can't understand the instructions http://wiki.languagetool.org/finding-errors-using-n-gram-data
```
java -cp "$HOME"/LanguageTool-4.9/languagetool-server.jar org.languagetool.server.HTTPServer --port 8081 --config "$HOME"/LanguageTool-4.9/languagetool.cfg
```
(and the file it points to is)
```
languageModel=/home/david/LanguageTool-4.9
```
My `LanguageTool-4.9` directory looks like this:
```
├── en
├── libs
├── META-INF
├── org
├── third-party-licenses
├── languagetool.cfg
├── languagetool.jar
├── languagetool-server.jar
├── languagetool-commandline.jar
├── README.md
├── CHANGES.md
├── testrules.sh
├── testrules.bat
├── COPYING.txt
└── CHANGES.txt
```
The n-grams are inside `en`. 

Do I NEED a config file? Do I need to set a language, if so, on the command line? If would be more helpful if Language Tool gave errors and told the user if it was using the n-grams rather than being a mystery that needs a test file to be run.

Please supply an example of how it works, it is a great program, THANKS!

**Comment by danielnaber** [comment]
Please try with a recent snapshot (https://internal1.languagetool.org/snapshots/), there was a bug with ngrams in 4.9.

**Comment by David-Else** [comment]
@danielnaber If I use the new `LanguageTool-20200426-snapshot` version it partly works, I get the bold highlighted in the following sentences when using the server:

I can't remember how to go **their**. I do not now where it comes from.

But if I use the latest LibreOffice version with the same n-gram data I get correct correction of both errors:

I can't remember how to go **their**. I do not **now** where it comes from.

**Comment by danielnaber** [comment]
> But if I use the latest LibreOffice version with the same n-gram data I get correct correction of both errors:

I can't reproduce that. For me, "I do not now where it comes from." doesn't work in any case (it's not found due to the thresholds being more strict now).

**Comment by David-Else** [comment]
It seems like the main problem here is the WIKI docs, the actual error is fixed in the snapshot. May I propose:

1. You re-write the following line to be more clear, and include a working example after it:
```
Server mode: Start with the --languageModel option. Alternatively, you can start with the --config file option. This properties file needs to have a languageModel=… entry pointing to the ngram-index directory. Using the properties files will give you some advanced configurations. Call java -jar languagetool-server.jar to get a list of all options.
```
2. Remove `I don't now where it comes from` as an example of it working, as this is no longer the case.
3. You push out a patch to fix the n-gram bug in 4.9, seems like a long time to wait until 5.0 for it to work.

Feel free to close this issue as you see fit, cheers.

**Comment by danielnaber** [comment]
There's now a 4.9.1 release candidate at https://languagetool.org/download/LanguageTool-4.9.1-rc1.zip. Could you see if it works for you?

**Comment by David-Else** [comment]
I just tested the rc1, and am pleased to report it now finds `their` and `now`  in `I can't remember how to go their. I do not now where it comes from` :)

**Comment by EricTendian** [comment]
Looks like the release candidate was promoted to a regular release, so can this issue be closed now? I just closed my earlier issue #2624 because I had the same problem but it's fixed.
